ci/build-project.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+UNREAL_PATH=${1}
+TEST_REPO_BRANCH=${2}
```

Comment:
```suggestion
TEST_REPO_BRANCH="${2:?Please enter a test repo branch}"
```

https://stackoverflow.com/questions/8889302/what-is-the-meaning-of-a-question-mark-in-bash-variable-parameter-expansion-as-i

Reply:
ohh, that is useful